interface/modules/zend_modules/module/Installer/src/Installer/Controller/InstallerController.php

```diff
@@ -327,4 +432,101 @@ public function nickNameAction()
         echo $this->getInstallerTable()->validateNickName(trim($nickname));
         exit(0);
     }
+
+    function getModuleVersionFromFile($modId)
+    {
+        //SQL version of Module
+        $dirModule = $this->getInstallerTable()->getRegistryEntry($modId, "mod_directory");
+        $ModulePath = $GLOBALS['srcdir']."/../".$GLOBALS['baseModDir']."zend_modules/module/".$dirModule -> modDirectory;
+
+        $version_of_module = $ModulePath."/version.php";
+        $table_sql = $ModulePath."/table.sql";
+        $install_sql = $ModulePath."/sql/install.sql";
```

Comment:
What is the difference between table.sql and sql/install.sql ?

Reply:
Compatibility with functionality currently available